### PR TITLE
Update to reflect current operations

### PR DESCRIPTION
**Pull request by andrespavez**

### CODE REVIEW

**Comment by paulehoffman** [comment]
These changes look fine to me. I have tagged @jschlyter for a second review, because he's part of the algorithm rollover design team that caused this first set of changes. If @ableyjoe or @GuillaumeBailey  want to comment, that's great, but given that this set of changes comes directly from IANA, I'd say we should go with them.

*NOTE* We need to hear from @andrespavez  before we actually accept this PR: he needs to clear it with other folks in IANA. But we can work on it until then.

**Review by jschlyter — APPROVED** [review]
These changes looks good to me, but my approval **is not** authoritative. Looking forward for confirmation by IANA.

File: draft-bash-rfc7958bis.mkd
Comment (kjd) [inline draft-bash-rfc7958bis.mkd]: Is the mention of PTI germane to the document? I'd actually suggest going in the opposite direction, i.e. ".. is an IANA function." without mention of ICANN or PTI.

File: draft-bash-rfc7958bis.mkd
Comment (kjd) [inline draft-bash-rfc7958bis.mkd]: It doesnt make sense to have a list with a single item, suggest "IANA publishes trust anchors for the root zone as an XML document that contains the hashes of the DNSKEY records."

**Review by kjd — CHANGES_REQUESTED** [review]
See comments inline.

**Review by ableyjoe — COMMENTED** [review]
If the goal of this document is to document what is actually done (i.e. reflect reality) and the new text is accurate in that sense and achieves that goal, then I have no problem with it.

I do think that the reliance on an unsigned data structure ought to trigger questions during the review cycle. (Doesn't this mean that DNSSEC is only as secure as the weakest of the root CA used to establish the TLS session and the CDN used to terminate it? Aren't we reducing the security in the system by eliminating more secure methods of retrieving the trust anchor set in favour of one that has flaws?)

I do not necessarily think those questions need to be anticipated and answered in this document, but it's perhaps worth more than a moment's thought. People are known to be sensitive to appearances of centralisation, even if TOFU is the practical reality of how this works today.

**Comment by kjd** [comment]
I think the baseline is to update the document to current reality, but I agree with Joe that the status quo seems to lack rigour and perhaps the next step is to work out what needs to be added (both to the document and operationally) to address any gaps.

File: draft-bash-rfc7958bis.mkd
Comment (andrespavez) [inline draft-bash-rfc7958bis.mkd]:
```suggestion
IANA function. A detailed description of
```

File: draft-bash-rfc7958bis.mkd
Comment (andrespavez) [inline draft-bash-rfc7958bis.mkd]:
```suggestion
IANA publishes trust anchors for the root zone as an XML document that contains the hashes of the DNSKEY records.
```

File: draft-bash-rfc7958bis.mkd
Comment (andrespavez) [reply draft-bash-rfc7958bis.mkd]: Ok

File: draft-bash-rfc7958bis.mkd
Comment (andrespavez) [reply draft-bash-rfc7958bis.mkd]: ok

**Review by andrespavez — COMMENTED** [review]
Suggested changes by @kjd have been addressed

**Comment by jschlyter** [comment]
> I do think that the reliance on an unsigned data structure ought to trigger questions during the review cycle. (Doesn't this mean that DNSSEC is only as secure as the weakest of the root CA used to establish the TLS session and the CDN used to terminate it? Aren't we reducing the security in the system by eliminating more secure methods of retrieving the trust anchor set in favour of one that has flaws?)

One can currently verify the authenticity and integrity of the trust anchor set in multiple ways, at least the following:

1. Verify the [detached S/MIME signature](http://data.iana.org/root-anchors/root-anchors.p7s) (from the IANA PKI)
2. Trust the TLS connection for fetching the XML (based on WebPKI)
3. Use a Trust Anchor set fetched by the vendor and verified OOB with IANA

The above could be augmented with more mechanisms, e.g. additional signatures from other parties, if required. Given how the validation software itself is distributed, I would say this is quite reasonable. Having said that, I do think we should encourage other mechanisms as well.

**Comment by paulehoffman** [comment]
I am going to accept the changes so we have a cleaner doc to work from.

**Comment by paulehoffman** [comment]
Annnnd, I should not have done that. I didn't realize that review changes were not applied. I will fix this during the week.

**Pull request merged**
